### PR TITLE
Add api calls

### PR DESCRIPTION
**Pull request by burkegg**

### CODE REVIEW

File: client/ReviewModule.jsx [hunk 1]
```diff
@@ -40,19 +30,26 @@ export default class ReviewModule extends React.Component {
...
-      console.log(searchText);
-    });
+    this.setState({ searchText: e.target.value });
   }
 
   avgStars() {
-    // Get all the stars in each review. avg them. return the avg.
     let sum = 0;
     const { reviews } = this.state;
     sum = reviews.reduce((acc, curr) => (acc + curr.stars), 0);
```
Comment (jmbice) [inline client/ReviewModule.jsx:55]: Should you be returning sum?

File: client/ReviewModule.jsx [hunk 2]
```diff
@@ -1,36 +1,26 @@
...
+          locname: 'Intial',
           stars: 3,
           posted: '2018-12-25',
           username: 'Tim, the user',
-          userLoc: 'San Francisco, CA',
+          userloc: 'San Francisco, CA',
           numfriends: 15,
           photoLoc: '../photos/p1.jpg',
-          mumReviews: 5,
+          mumreviews: 5,
```
Comment (jmbice) [inline client/ReviewModule.jsx]: Is this supposed to be 'mumreviews'? Not sure what 'mum' refers to, or why it is not camelCase. Just wanted to make sure you're certain!

File: database/datagenerator.js [hunk 3]
```diff
@@ -151,6 +151,7 @@ const reviews = function generateReviews(reviewers, restaurants, paragraphs) {
     })
```
Comment (jmbice) [inline database/datagenerator.js:151]: Not 100% sure what the closing parens and curly bracers on lines 151-153 are about, but perhaps I can't see their starting points.

Comment (burkegg) [reply database/datagenerator.js:151]: Those go with the nested forEach's.

Comment (jmbice) [reply database/datagenerator.js:151]: Ha, ha! I see it now! Github had minimized the code so I couldn't see. My bad!

**Review by jmbice — APPROVED** [review]
Excellent work as always!

**Pull request merged**